src/main/java/com/ghilly/service/CountryServiceRest.java

```diff
@@ -0,0 +1,41 @@
+package com.ghilly.service;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CountryServiceRest implements CountryService {
+
+    private static final Logger logger = LoggerFactory.getLogger(CountryServiceRest.class);
+
+
+    @Override
+    public String add(String countryName) {
+        logger.info("The country {} was added by service.", countryName);
+        return countryName;
+    }
+
+    @Override
+    public List<String> receiveList() {
+        logger.info("receiveList");
+        return new ArrayList<>();
+    }
+
+    @Override
+    public String receiveCountry(int countryId) {
+        return "receive" + countryId;
+    }
+
+    @Override
+    public String upgrade(int countryId, String newCountryName) {
+        logger.info("The country with ID {} was upgraded, new name is {}.", countryId, newCountryName);
+        return newCountryName;
+    }
+
+    @Override
+    public String remove(int countryId) {
+        return "clear" + countryId;
```

Comment:
Why do you use logger in some methods but not here?

Reply:
Done.